bond/__init__.py

```diff
@@ -19,6 +19,9 @@
                    BOND_DEVICE_ACTION_TURN_LIGHT_ON,
                    BOND_DEVICE_ACTION_TURN_LIGHT_OFF,
                    BOND_DEVICE_ACTION_TOGGLE_LIGHT,
+                   BOND_DEVICE_ACTION_START_DIMMER,
+                   BOND_DEVICE_ACTION_START_UP_LIGHT_DIMMER,
```

Comment:
Are these the names bond gives us? If not can we use Increase and Decrease terminology.

Reply:
Yes they are...please see [this section of the API](http://docs-local.appbond.com/#section/Features/Misc-including-dimmers).  Note that this is for dimmers that just cycle from dim to bright on a single button...not for discrete increase/decrease buttons.  The "up" reference in the code is for lights that shine "up" to the ceiling, as opposed to down - not referring to the brightness.